html/settings/_settings.scss

```diff
@@ -1670,6 +1670,18 @@ $sprk-masthead-accordion-item-open-line-height: 2px !default;
 /// The padding of the items in the
 /// narrow viewport Masthead Accordion.
 $sprk-masthead-accordion-item-padding: $sprk-space-m !default;
+/// The text alignment of an item in the
```

Comment:
Is this a navigation item? Should add that its navigation

Reply:
corrected for clarity!